tests/sentry/options/test_manager.py

```diff
@@ -106,6 +106,26 @@ def test_legacy_key(self):
         assert self.manager.delete("sentry:foo")
         assert self.manager.get("sentry:foo") == ""
 
+    def test_legacy_url_prefix_key_use_db_setting(self):
+        """
+        TODO: Remove when SENTRY_URL_PREFIX is completely deprecated
+        """
+        self.manager.register("system.url-prefix", flags=FLAG_PRIORITIZE_DISK)
+        with patch.object(self.store.cache, "get", return_value="https://sentry.example.com"):
+            with self.settings(SENTRY_OPTIONS={}):
+                assert self.manager.get("system.url-prefix") == "https://sentry.example.com"
+                assert settings.SENTRY_URL_PREFIX == "https://sentry.example.com"
+                assert settings.SENTRY_OPTIONS["system.url-prefix"] == "https://sentry.example.com"
+
+    def test_legacy_url_prefix_use_config_setting(self):
+        """
+        TODO: Remove when SENTRY_URL_PREFIX is completely deprecated
+        """
+        self.manager.register("system.url-prefix", flags=FLAG_PRIORITIZE_DISK)
+        with patch.object(self.store.cache, "get", return_value="https://sentry.example.com"):
+            with self.settings(SENTRY_OPTIONS={"system.url-prefix": "https://sentry.updated.com"}):
+                assert self.manager.get("system.url-prefix") == "https://sentry.updated.com"
```

Comment:
Can we include `settings.SENTRY_URL_PREFIX` and `settings.SENTRY_OPTIONS` checks in here as we do above?